### PR TITLE
main-content

### PR DESCRIPTION
**Pull request by coder-mezab**
completed grid mansonry

### CODE REVIEW

**Review by faran4engg — COMMENTED** [review]
Live link ?

File: style.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  box-sizing: border-box;
+}
+body {
+  background: #000;
+}
+
+.big-container {
+  display: flex;
+
+  justify-content: center;
```
Comment (faran4engg) [inline style.css:13]: these 2 are related. 
cc/ @anask601

**Review by faran4engg — APPROVED** [review]
NIce one... left a minor comment ! I am approving now.. but please fix it first and then merge !

File: style.css (hunk 1 above)
Comment (coder-mezab) [reply style.css:13]: yes @faran4engg

**Review by faran4engg — CHANGES_REQUESTED** [review]
check responsiveness !
@coder-mezab 
.
.
Screen Shot 2021-10-05 at 10 05 25 AM

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
Read code

File: style.css (hunk 1 above)
Comment (faran4hatch) [reply style.css:15]: Group it then na :D

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>css-grid-layout</title>
+  </head>
+  <div>
```
Comment (faran4engg) [inline index.html:10]: wow, so you removed `body` LOL

**Comment by coder-mezab** [comment]
all done @faran4engg

File: index.html (hunk 2 above)
Comment (faran4engg) [reply index.html:11]: @coder-mezab ??

**Review by faran4hatch — APPROVED** [review]

**Pull request merged**